app/src/main/java/com/arqoders/hotelapp/data/server/TheEntityDbService.kt

```diff
@@ -0,0 +1,16 @@
+package com.arqoders.hotelapp.data.server
```

Comment:
El nombre del archivo debería coincidir con el nombre de la interface para que sea localizada desde fuera sin tener que ver el archivo.

Reply:
Renamed to ApiService